shub/config.py

```diff
@@ -28,6 +28,8 @@ def __init__(self):
         }
         self.apikeys = {}
         self.version = 'AUTO'
+        self.stacks = {}
+        self.requirements_file = None
```

Comment:
Yes, it will cause panic on existent projects that suddenly after upgrading shub will start installing more dependencies than used to be.

I think we can default it to another more uncommon file, like `requirements-scrapinghub.txt`, unlikely it exists and you can pin dependencies in more detail while still can use requirements.txt for local development.